pkgs/development/node-packages/composition-v13.nix

```diff
@@ -2,7 +2,7 @@
 
 {pkgs ? import <nixpkgs> {
     inherit system;
-  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-13_x"}:
+  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-8_x"}:
```

Comment:
This shouldn't have been committed right?

Reply:
Thanks, the script must have done this but I reverted the change